packages/contracts-ops/contracts/test/RebootReplica.t.sol

```diff
@@ -15,11 +15,13 @@ contract ReplicaRebootTest is RebootTest, ReplicaTest {
         replica = ReplicaHarness(
             address(getReplicaOf(localDomainName, remote))
         );
-        assertEq(replica.updater(), updaterAddr);
         upgradeReplicaHarness();
+        // Replica test vars
         committedRoot = replica.committedRoot();
         optimisticTimeout = replica.optimisticSeconds();
         setUpBadHandlers();
+        // check fork setup
+        assertEq(replica.updater(), updaterAddr);
```

Comment:
no asserts in setup

Reply:
https://github.com/nomad-xyz/monorepo/pull/545/commits/a50e36eac21055df15997d24735ae1ce33d63b64